docs/reference/event_handlers/index.md

```diff
@@ -1,20 +1,47 @@
-# EventHandler
+# EventHandler Base Class
 
 db-ally provides an `EventHandler` abstract class that can be used to log the runs of db-ally to any external systems.
 
+!!! tip
+    To learn how to create a cutom `EventHandler`, visit [How-To: Create your own event handler](../../how-to/create_custom_event_handler.md).
+
+
+
 
 ## Lifecycle
 
 
-Every run of db-ally will trigger any instance of EventHandler that was registered using `dbally.use_event_handler` method in following manner:
+Every run of [dbally.Collection.ask](../collection.md/#collection.ask) will trigger every instance of EventHandler that was registered using [`dbally.use_event_handler`](../index.md/#dbally.use_event_handler) method in following manner:
```

Comment:
The sentence structure seems a bit complicated. Maybe "Each run of ... will trigger all instances of ..."?